apps/hash-api/src/graphql/resolvers/integrations/linear/sync-workspaces-with-teams.ts

```diff
@@ -46,15 +40,15 @@ export const syncLinearIntegrationWithWorkspacesMutation: ResolverFn<
     },
   );
 
-  const userAccountId = extractOwnedByIdFromEntityId(
+  const ownedById = extractOwnedByIdFromEntityId(
```

Comment:
See other comment - anywhere in the Linear code it's called `userAccountId` the intention is that it's specifically for a user, not an org

Reply:
Fixed in [`be38bcf`](https://github.com/hashintel/hash/pull/3128/commits/be38bcf41984e5203d2da4bacfa592957959ec09)